core/src/test/fixtures_postgresql/returning/Sample.s

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE account(
+    id      SERIAL PRIMARY KEY,
+    name    TEXT NOT NULL,
+    balance INT  NOT NULL
+);
+
+INSERT INTO account(name, balance)
+VALUES ('Jane Doe', 0), ('John Doe', 0)
+RETURNING id;
```

Comment:
Not the most useful `RETURNING` clauses in this file, as we don't read the output of `INSERT` in SqlDelight, but it is nevertheless valid PostgreSQL. This PR is mostly just a precursor to https://github.com/AlecStrong/sql-psi/issues/162.